Library/Homebrew/formula_installer.rb

```diff
@@ -589,7 +589,7 @@ def build
       -I #{HOMEBREW_LOAD_PATH}
       --
       #{HOMEBREW_LIBRARY_PATH}/build.rb
-      #{formula.path}
+      #{formula.alias_path || formula.path}
```

Comment:
Think this logic may be worth pulling into a formula method that just does this `||`. `alias_path_or_path` or something with a better name?